lib/muser_ctx.c

```diff
@@ -1425,6 +1411,39 @@ int lm_setup_device_irq_counts(lm_ctx_t *lm_ctx,
     return 0;
 }
 
+int lm_setup_device_migration(lm_ctx_t *lm_ctx, lm_migration_t *migration)
+{
+    lm_reg_info_t   *migr_reg;
+    int ret = 0;
+
```

Comment:
validate `lm_ctx` and `migration` arguments

Reply:
Added fixme. As mentioned in the introduction of the PR, arg validation will be done later.